.github/workflows/bosh-release.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - name: Check out the codebase.
         uses: actions/checkout@v3
       - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@0c611abec1fc679030907286a2c084cc92fb8f2e
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
```

Comment:
we use a commit id to avoid getting hacked if someone pushes a malicious commit to the master branch, can we update to the latest commit id if needed?